lib/routes/create_invoice/create_invoice_page.dart

```diff
@@ -198,9 +198,13 @@ class CreateInvoicePageState extends State<CreateInvoicePage> {
     final accountBloc = context.read<AccountBloc>();
     final currencyBloc = context.read<CurrencyBloc>();
 
+    final lspInfo = context.read<LSPBloc>().state?.lspInfo;
+    final cheapestFeeParams = lspInfo?.openingFeeParamsList.values.first;
```

Comment:
This seems to work but I think it is better to have one stream of the LSP that wraps the build method and pass the fee params explicitly to this `_createInvoice` function. 
This fee params also should be passed to the `ReceivableBTCBox`.
This way we are sure that the same fee params that are displayed to the user are sent to the create invoice and it is trivial to understand that from the code.

Reply:
we can use context.watch which listens to changes to <Block>().state. Though we need to define it earlier in the build method so that it rebuilds if there are any changes.